.ci_tools/build_samples.sh

```diff
@@ -77,6 +78,7 @@ declare apks=(
   native-media/app/build/outputs/apk/debug/app-debug.apk
   native-plasma/app/build/outputs/apk/debug/app-debug.apk
   prefab/curl-ssl/app/build/outputs/apk/debug/app-debug.apk
+  prefab/prefab-publishing/mylibrary/build/outputs/aar/debug/mylibrary-debug.aar
```

Comment:
the real path does not have "debug" somehow, it is ( on Mac ):
  prefab/prefab-publishing/mylibrary/build/outputs/aar/mylibrary-debug.aar

may you double check on it?